src/toICN-after.js

```diff
@@ -36,10 +39,13 @@ if(detectedKey == ""){
   key = "";
   detectedKeyMinorSignature = "u";
 }
+else{
+  isKeyWritten = true;
+}
 
 let displayedKey = exports.DisplayedKey(detectedKey, detectedMinorSignature);
```

Comment:
すいませんmainにバグが混入しています・・このPRで直してもらえると・・

```suggestion
let displayedKey = exports.DisplayedKey(detectedKey, detectedKeyMinorSignature);
```